tony-core/src/main/java/com/linkedin/tony/tensorflow/TonySession.java

```diff
@@ -161,6 +163,43 @@ public boolean allTasksScheduled() {
     return true;
   }
 
+  public int getTotalTasks() {
+    return jobTasks.values().stream().mapToInt(arr -> arr.length).sum();
+  }
+
+  public int getTotalTrackedTasks() {
+    int count = 0;
+    for (Map.Entry<String, TonyTask[]> entry : jobTasks.entrySet()) {
```

Comment:
feels like this could be a simple 1 line reduce function

Reply:
Very well:
```
    return jobTasks.entrySet().stream().filter(entry -> Utils.isJobTypeTracked(entry.getKey(), tonyConf))
        .mapToInt(entry -> entry.getValue().length).sum();
```